dask/dataframe/tests/test_reshape.py

```diff
@@ -86,6 +86,9 @@ def test_get_dummies_kwargs():
     tm.assert_index_equal(res.columns, pd.Index([1, 2, 3, 5, np.nan]))
 
 
+@pytest.mark.filterwarnings(
+    "ignore:In a future version, passing a SparseArray:FutureWarning"
+)  # https://github.com/pandas-dev/pandas/issues/45618
```

Comment:
I find it a bit strange that this ignore is needed, as the example shouldn't be dealing with a sparse dtype in the index, only sparse columns.

Reply:
Ah, I see that's exactly why you opened the pandas issue :) https://github.com/pandas-dev/pandas/issues/45618